src/pages/index.js

```diff
@@ -65,25 +132,81 @@ function setFormInfo(nameSelector, detailsSelector) {
 }
 
 function createCard(data) {
-  const cardElement = new Card(data, handleImageClick , "#cards-template");
+  const cardElement = new Card(data, handleImageClick , "#cards-template" , handleDeleteClick, handleLikeClick);
   return cardElement.generateCard();
 }
 
+function handleLikeClick(card) {
+  if(card.isLiked) {
+    api.unlikeCard(card._id).then((res) => {
+      // update the UI
+
+      card.updateIsLiked(res.isLiked);
+    })
+  } else {
+    api.likeCard(card._id).then((res) => {
+      // update the UI
+      card.updateIsLiked(res.isLiked);
+    })
+  }
+}
+
 function handleImageClick(imgData) {
   previewModal.open(imgData);
 }
 
+function handleDeleteClick(card) {
+  console.log(card);
+  deleteCardModal.open();
+  deleteCardModal.setHandleConfirm(() => {
+    return api
+      .deleteCard(card._id)
+      .then(() => {
+        card._handleDeleteButton();
```

Comment:
Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.